apps/hash-api/src/graphql/resolvers/ontology/entity-type.ts

```diff
@@ -85,7 +84,7 @@ export const getAllLatestEntityTypesResolver: ResolverFn<
     },
   });
 
-  return mapSubgraph(entityTypeSubgraph);
+  return entityTypeSubgraph as Subgraph;
```

Comment:
Not strictly needed here but let's specify the root types (you'll need to update the imports). Ideally we're going to do one of the TODOs in the code that say we should capture these root types in the GraphQL types, and this should mean we've already done part of that:
```suggestion
  return entityTypeSubgraph as Subgraph<EntityTypeRootType>;
```

Although really we should be using the API defined in the `hash-api/src/graph/ontology/primitive/entity-type`, but I just had a look and we don't actually expose the `getEntityTypesByQuery` endpoint yet 🙃 

I'll probably fix this when I reimplement 'aggregations' (now called queries)

Reply:
Thanks for the heads-up! As we have the information, we should specify the root type. I applied your suggestion in [Apply suggestions from Code Review](https://github.com/hashintel/hash/pull/2110/commits/460911c284da89861487b718d9a50812321df97f)